tests/src/test/java/alluxio/cli/fs/command/CopyFromLocalCommandIntegrationTest.java

```diff
@@ -45,105 +45,6 @@
   @Rule
   public TemporaryFolder mTestFolder = new TemporaryFolder();
 
-  @Test
```

Comment:
Is there anyway to keep these tests?

Reply:
I don't see a simple way to change these tests so that they don't leak files.